docker-compose.yml

```diff
@@ -10,9 +8,16 @@ app:
     - AWS_ACCESS_KEY
     - AWS_SECRET_KEY
   links:
-    - rabbitmq
-    - datomic
+    - wildfly
 rabbitmq:
   image: rabbitmq:3.5.0-management
-datomic:
-  image: quay.io/democracyworks/datomic-tx:0.9.5153
```

Comment:
Does it make sense to run a datomic transactor here? How does the transactor relate to the USPS_DATOMIC_URI? Why not just run a datomic:mem?